test/inductor/test_aot_inductor.py

```diff
@@ -189,6 +190,26 @@ def forward(self, x, y):
                 model, example_inputs, "AOTInductorModelRunMinimalArrayrefInterface(", 1
             )
 
+    def test_triton_kernel_bool_param(self):
```

Comment:
does this test repro in normal inductor or only with aot inductor? If so is it possible to put it in the normal inductor tests & let it run via the cpp_wrapper tests?

Reply:
Added to test_torchinductor.py